README_python.md

```diff
@@ -1,17 +1,46 @@
 # Halide Bindings for Python
```

Comment:
We should consider changing "you" to "one" throughout this document.

Reply:
I don't disagree, but it's not clear to me what the appropriate phrasing becomes in all such cases -- sometimes it becomes awkwardly passive-voice, whereas the 'you/your' phrasing doesn't (at least to my ear). Do you see this as simply a matter of preferred style, or are there other correctness issues I'm missing?

Reply:
I'm happy with this document using the second person. "One" sounds excessively formal to my ear. Comparable documentation from other languages uses similar second person wording, see, e.g. the Rust Book https://doc.rust-lang.org/book/ch03-04-comments.html